src/security/cert_generators/file/certificate_db.cc

```diff
@@ -107,12 +105,13 @@ Ssl::Locker::Locker(Lock &aLock, const char *aFileName, int aLineNo):
 
 Ssl::Locker::~Locker()
 {
+    (void)lineNo;
     if (weLocked)
         lock.unlock();
 }
 
 Ssl::CertificateDb::Row::Row()
-    :   width(cnlNumber)
+    : width(cnlNumber)
```

Comment:
An out of scope change? If you insist on polishing this, please keep `:` on the first line. Personally, I would place the data member initialization on the same line as well in this simple case.

Reply:
missed. Moving on single previous line. I don't recall introducing the change, it may have been astyle; in this case it will be reintroduced